hunts/views.py

```diff
@@ -173,10 +173,14 @@ def post(self, request, hunt_slug):
             if google_api_client:
                 google_api_client.add_puzzle_link_to_sheet(puzzle_url, sheet)
 
-            chat_room = ChatRoom.objects.create(
-                service=settings.CHAT_DEFAULT_SERVICE, name=name
-            )
-            chat_room.create_channels()
+            if settings.CHAT_DEFAULT_SERVICE is not None:
```

Comment:
nit: Can we simplify this to just `if settings.CHAT_DEFAULT_SERVICE:`? It's a bit nicer to read and also catches the empty string case.